frontend/src/stimulus/controllers/dynamic/meeting-section-form.controller.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * -- copyright
+ * OpenProject is an open source project management software.
+ * Copyright (C) 2023 the OpenProject GmbH
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License version 3.
+ *
+ * OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+ * Copyright (C) 2006-2013 Jean-Philippe Lang
+ * Copyright (C) 2010-2013 the ChiliProject Team
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * See COPYRIGHT and LICENSE files for more details.
+ * ++
+ */
+
+import * as Turbo from '@hotwired/turbo';
+import { Controller } from '@hotwired/stimulus';
+
+export default class extends Controller {
+  static values = {
+    cancelUrl: String,
+  };
+
+  declare cancelUrlValue:string;
+
+  connect():void {
+    this.focusInput();
+  }
+
+  focusInput():void {
+    const titleInput = this.element.querySelector('input[name="meeting_section[title]"]');
+
+    setTimeout(() => {
+      this.element.scrollIntoView({ block: 'center' });
+        (titleInput as HTMLInputElement).focus();
+        this.setCursorAtEnd(titleInput as HTMLInputElement);
+    }, 100);
```

Comment:
As the titleInput is already present, why did you add such a high delay? Do we need it at all?

Reply:
True, we don't need it - I took the implementation from the `agenda-item-form.controller` where it was required at some earlier point during implementation. It can be removed there as well as far as I tested it. Will do so